client/view/DocSection/docSection.js

```diff
@@ -0,0 +1,14 @@
+export function initMobileDocSections() {
+  document.querySelectorAll('section.DocSection').forEach((elem) => {
+    let openButton = document.createElement('button');
+    openButton.type = 'button';
+    openButton.innerHTML = 'open';
+    openButton.classList.add('DocSection__button');
+
+    openButton.addEventListener('click', () => {
+      elem.classList.toggle('DocSection--open');
+    })
+
+    elem.firstElementChild.insertBefore(openButton, elem.firstElementChild.firstElementChild)
```

Comment:
`firstElementChild.firstElementChild`

It's hard to understand. Let's put it in a variable with the name.